app/controllers/executions_controller.rb

```diff
@@ -14,6 +14,7 @@ def index
 
   # GET /executions/1
   def show
+    @measures  = @execution.measures.reorder(created_at: :desc).limit 20
```

Comment:
Acá hay otros espacios innecesarios:

```suggestion
    @measures = @execution.measures.reorder(created_at: :desc).limit 20
```